packages/frames.js/package.json

```diff
@@ -187,6 +187,16 @@
         "default": "./dist/farcaster-v2/types.cjs"
       }
     },
+    "./farcaster-v2/verify-app-key-with-neynar": {
+      "import": {
+        "types": "./dist/farcaster-v2/verify-app-key-with-neynar.d.ts",
+        "default": "./dist/farcaster-v2/verify-app-key-with-neynar.js"
+      },
+      "require": {
+        "types": "./dist/farcaster-v2/verify-app-key-with-neynar.d.cts",
+        "default": "./dist/farcaster-v2/verify-app-key-with-neynar.cjs"
+      }
+    },
     "./core": {
```

Comment:
i'm a bit hesitant about this export path name. should we not just export it as `/farcaster-v2/neynar` or `farcaster-v2/verify`

Reply:
renamed to `verify`